araig_utils/launch/export_video.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# variables for all bag files
+BAG[0]=test3_2021-09-15-15-06-40.bag
```

Comment:
hardcoded values in the script? This will not work for other test cases then? Please redo this.